### DISCUSSION TITLE
More atom types for the same element

### DISCUSSION THREAD

**Discussion by martino95** [deepmodeling/deepmd-kit#247] [Q&A] [answered]
Hello,

I'm developing a coarse grained model of nucleobases in water. I'm applying the two step procedure described in the DeepCG paper: currently I'm working on a deepMD all atom model that I'll then use to generate the dataset to learn the finale CG model.
Thus I'm not interested in the transferability of the deepMD model to different molecules, nor in describing chemical reactions or particular changes in the molecular structure: I only need a fast way to obtain a reliable sampling of the phase space of the system at (nearly) QM accuracy.
My question is: is it worth to define (as in classical FF models) different atom types for different atoms of the same element?
For example: Adenosine monophosphate has many different O atoms: an etheric one, an esteric one two hydroxylic... should I define one atom type for each of these different "kinds" of oxygen? Basing on your experience would it bring a significant efficiency improvement?

Kindly,
Riccardo

**Comment by jameswind** [comment, marked as answer]
A short answer is no, since it will introduce much more parameters in the
DeePCG model, and the difference of subtypes of atoms with the same type
should be signaled by the local chemical environment and should be captured
by the same set of parameters. You may try to do the fitting and see.
